src/applications/disability-benefits/all-claims/tests/data/transformed-data/maximal-test.json

```diff
@@ -0,0 +1,200 @@
+{
+  "form526": {
```

Comment:
The json files in `test/data/transformed-data/` correspond to what was in `test/schema/transformedData.js`, but now we can dynamically require them. I didn't actually change the existing transformed data.